### PR TITLE
3 0 autogen

### PR DESCRIPTION
**Pull request by DenverM80**

### CODE REVIEW

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/UsersAndGroups_Test.java [hunk 1]
```diff
@@ -739,22 +740,13 @@ public void putGlobalBucketAclForGroupUUID() throws IOException, SignatureExcept
...
-            final DeleteBucketAclSpectraS3Response deleteBucketAclSpectraS3Response = client
-                    .deleteBucketAclSpectraS3(new DeleteBucketAclSpectraS3Request(
-                            putBucketAclForUserSpectraS3Response.getBucketAclResult().getId().toString()));
-
-            assertThat(deleteBucketAclSpectraS3Response.getStatusCode(), is(204));
-
-        } finally {
-            if (putBucketAclForUserSpectraS3Response != null) {
-                client.deleteBucketAclSpectraS3(new DeleteBucketAclSpectraS3Request(
+        putBucketAclForUserSpectraS3Response = client.putBucketAclForUserSpectraS3(
```
Comment (RachelTucker) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/UsersAndGroups_Test.java]: There is no need for the null initialization. These two lines can be combined, and the variable putBucketAclForUserSpectraS3Response can then be made final.

File: ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/UsersAndGroups_Test.java [hunk 2]
```diff
@@ -738,23 +726,13 @@ public void putGlobalBucketAclForGroupUUID() throws IOException, SignatureExcept
...
-            final DeleteBucketAclSpectraS3Response deleteBucketAclSpectraS3Response = client
-                    .deleteBucketAclSpectraS3(new DeleteBucketAclSpectraS3Request(
-                            putBucketAclForUserSpectraS3Response.getBucketAclResult().getId().toString()));
-
-            assertThat(deleteBucketAclSpectraS3Response.getStatusCode(), is(204));
-
-        } finally {
-            if (putBucketAclForUserSpectraS3Response != null) {
-                client.deleteBucketAclSpectraS3(new DeleteBucketAclSpectraS3Request(
+        PutBucketAclForUserSpectraS3Response putBucketAclForUserSpectraS3Response = client.putBucketAclForUserSpectraS3(
```
Comment (rpmoore) [inline ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/UsersAndGroups_Test.java]: Missing final

**Comment by rpmoore** [comment]
Does this change need to be merged to 3.2?

**Comment by RachelTucker** [comment]
That test file does not appear to be in 3.2

**Pull request merged**